presentation-compiler/src/main/dotty/tools/pc/completions/CompletionAffix.scala

```diff
@@ -1,56 +1,95 @@
 package dotty.tools.pc.completions
 
+import org.eclipse.lsp4j.TextEdit
+import org.eclipse.lsp4j.Position
+
 /**
  * @param suffixes which we should insert
+ * @param prefixes which we should insert
  * @param snippet which suffix should we insert the snippet $0
  */
 case class CompletionAffix(
-    suffixes: Set[SuffixKind],
-    prefixes: List[PrefixKind],
-    snippet: SuffixKind,
+    suffixes: Set[Suffix],
+    prefixes: Set[Prefix],
```

Comment:
`prefix: Prefix` ? In `loopPrefix` you take the head anyway.

Reply:
I've changed it to now properly loop. In the future I'd want to migrate all prefixes like access modifiers, override etc to this but it was not in scope of this PR